### ISSUE TITLE
Accessibility: Tooltips do not appear when using keyboard navigation

### ISSUE THREAD

**Issue by BenBeckley** [microsoft/vscode#242621]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 
- OS Version: 

We received this issue from accessbility testing of our VS Code extension. It is also reproducible using [vscode-extension-samples/tree-view-sample](https://github.com/microsoft/vscode-extension-samples/tree/main/tree-view-sample). The issue is essentially that treeitem tooltips are not being shown when using keyboard navigation.

Steps to Reproduce:

1. Run the tree-view-sample
2. Open a workspace that contains node packages in the Explorer
3. Open the Package Explorer side bar and refresh
4. Now the treeview should be populated
5. Use keyboard navigation (Tab, arrow keys) to navigate through the treeview elements

Expected: The tooltips should appear while using keyboard navigation.
Actual: The tooltips do not appear on keyboard navigation, but do appear on mouse hover.

**Comment by meganrogge** [comment]
Thanks for the issue - can you link the one from the testing?

**Comment by BenBeckley** [comment]
I can link the repro steps and instructions if need be, and it will be the same as what I shared with vscode-extension-sample from above. Here is the Microsoft Accessibility Standard for keyboard accessibility: [MAS 2.1.1 – Keyboard](https://aka.ms/MAS2.1.1).

Is the sample not sufficient to repro this issue?

**Comment by Fun678** [comment]
[](url)
